pkg/middleware/enforce_identity_test.go

```diff
@@ -81,26 +79,24 @@ func TestWrapMiddlewareWithSkipper(t *testing.T) {
 		return c.String(http.StatusOK, string(body))
 	}
 	e.GET("/ping", h)
-	e.GET(urlPrefix+"/v1/ping", h)
-	e.GET(urlPrefix+"/v1.0/ping", h)
+	e.GET("/ping/", h)
 	e.GET(urlPrefix+"/v1/repository_parameters/", h)
 
 	// A Success request to /ping family path
 	listSuccessPaths = []string{
 		"/ping",
-		urlPrefix + "/v1/ping",
```

Comment:
removed these because they don't seem to be registered routes, curling results in a 404 on main with or without an identity header

Reply:
errr but now in this PR any unregistered route returns a 400 instead of a 404 🤔

Reply:
that should be fixed [here](https://github.com/content-services/content-sources-backend/pull/911/commits/a4eefc4964d69863610be6ce9b9113235c0b0415)